build.gradle

```diff
@@ -349,6 +349,13 @@ task allJavadoc(type: Javadoc, group: "Documentation") {
         classpath += configurations.javacJar
     }
     doLast {
+        exec {
+            // Javadoc for to com.sun.tools.java.* is not publicly available, so these links are broken.
+            // This command removes those links.
+            executable "${plumeScriptsHome}/preplace"
+            args += ['-debug', '<a href="https://docs.oracle.com/javase/9/docs/api/com/sun/tools/javac/.*?>(.*?)</a>', '\\1', "${rootDir}/docs/api"]
```

Comment:
I don't think the `-debug` argument is necessary.
Also, this command only needs to be run in `api` directories.